src/port1.0/portutil.tcl

```diff
@@ -2579,8 +2579,8 @@ proc PortGroup {group version} {
         uplevel "source $groupFile"
         ui_debug "Sourcing PortGroup $group $version from $groupFile"
     } else {
-        lappend PortInfo(portgroups) [list $group $version ""]
-        ui_warn "PortGroup ${group} ${version} could not be located. ${group}-${version}.tcl does not exist."
+        ui_error "${subport}: PortGroup ${group} ${version} could not be located. ${group}-${version}.tcl does not exist."
+        return -code error "Invalid PortGroup found"
```

Comment:
I would rather say "PortGroup not found". We don't know for certain that it's invalid; the file could just be missing.